packages/components/ovh-at-internet/src/ovh-at-internet.ts

```diff
@@ -152,7 +152,7 @@ export default class OvhAtInternet extends OvhAtInternetConfig {
       debug(`tracking init: no backend (consent=${withConsent})`);
       this.tag = { debug: true };
     }
-    return Promise.resolve(withConsent ? this.initVisitorId() : null).then(() =>
+    Promise.resolve(withConsent ? this.initVisitorId() : null).then(() =>
       this.processTrackQueue(),
```

Comment:
```suggestion
    return Promise.resolve(withConsent ? this.initVisitorId() : null).then(() =>
      this.processTrackQueue(),
```
Why you remove the return ?

Reply:
it is asked by eslint.
![image](https://github.com/ovh/manager/assets/146735326/d8e6a1ee-06ef-4b03-992f-11eedaad3102)